app/Policies/EventPolicy.php

```diff
@@ -25,8 +27,15 @@ public function viewAny(?User $user): bool
         return true;
     }
 
-    public function view(?User $user): bool
+    public function view(?User $user, Event $event): bool
     {
+        if ($event->active_from != null && $event->active_from > Carbon::now()) {
```

Comment:
nit: Recommend using `!==` to enforce type safety.

Here's a contrived example:
```php
var_dump("" == null); // true ?!
var_dump("" === null); // false
```

Reply:
done